createdisk-library.sh

```diff
@@ -294,13 +294,11 @@ function download_podman() {
       mkdir -p podman-remote/mac
       curl -L https://github.com/containers/podman/releases/download/v${version}/podman-remote-release-darwin.zip -o podman-remote/mac/podman.zip
       ${UNZIP} -o -d podman-remote/mac/ podman-remote/mac/podman.zip
-      mv podman-remote/mac/podman-${version}/podman  podman-remote/mac
```

Comment:
Fwiw, I'd favour something like 
```
if [ -e podman-remote/mac/podman-${version}/podman ]; then
    mv podman-remote/mac/podman-${version}/podman  podman-remote/mac
fi
```
(or checking for the directory existence), but fine with me to go with this revert now, we can polish it when we need to re-revert